common/changes/office-ui-fabric-react/link_focus_outline_2019-01-16-00-08.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Added styling to Link to allow it to have a focus outline that doesn't clip when wrapping images.\"",
```

Just a suggestion 😄 ... no need to accept it.
```suggestion
      "comment": "Link: adds a focus outline that doesn't clip when wrapping images.",
```